frontend/src/metabase/timelines/common/components/MoveTimelineModal/MoveTimelineModal.tsx

```diff
@@ -33,8 +33,8 @@ const MoveTimelineModal = ({
       value={{ id: timeline.collection_id ?? "root", model: "collection" }}
       title={t`Move ${getTimelineName(timeline)}`}
       onClose={onClose}
-      onChange={async newCollection => {
-        await handleSubmit(newCollection.id);
+      onChange={newCollection => {
+        handleSubmit(newCollection.id);
```

Comment:
I think this will also prevent error propagation. we want this to throw.

#40700